webui/react/src/services/api-ts-sdk/api.ts

```diff
@@ -3152,6 +3152,199 @@ export const V1FittingPolicy = {
     PBS: 'FITTING_POLICY_PBS',
 } as const
 export type V1FittingPolicy = ValueOf<typeof V1FittingPolicy>
+/**
+ * Flat run respresentation.
+ * @export
+ * @interface V1FlatRun
+ */
+export interface V1FlatRun {
+    /**
+     * The id of the run.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    id: number;
+    /**
+     * The time the run was started.
+     * @type {Date | DateString}
+     * @memberof V1FlatRun
+     */
+    startTime: Date | DateString;
+    /**
+     * The time the run ended.
+     * @type {Date | DateString}
+     * @memberof V1FlatRun
+     */
+    endTime?: Date | DateString;
+    /**
+     * The current state of the run(trial).
+     * @type {Trialv1State}
+     * @memberof V1FlatRun
+     */
+    state: Trialv1State;
+    /**
+     * The tags of the run.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    tags: string;
+    /**
+     * The total size of checkpoints.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    checkpointSize: string;
+    /**
+     * The count of checkpoints.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    checkpointCount: number;
+    /**
+     * The type of searcher for the experiment.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    searcherType?: string;
+    /**
+     * The searcher metric name for the experiment.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    searcherMetric?: string;
+    /**
+     * Signed searcher metrics value.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    searcherMetricValue?: number;
+    /**
+     * The id of external run
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    externalRunId?: number;
+    /**
+     * Trial hyperparameters.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    hyperparameters?: string;
+    /**
+     * summary metrics.
+     * @type {any}
+     * @memberof V1FlatRun
+     */
+    summaryMetrics?: any;
+    /**
+     * The id of the experiment linked to the run.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    experimentId?: number;
+    /**
+     * The id of the user who created the run.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    ownerId?: number;
+    /**
+     * Original id of a forked or continued experiment.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    forkedFrom?: number;
+    /**
+     * The id of external experiment
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    externalExperimentId?: number;
```

Comment:
nit: experiments have `externalExperimentId` and `externalTrialId` as strings. i actually prefer numbers, but again consistency is good